pkg/tanka/parse.go

```diff
@@ -127,13 +127,23 @@ func evalJsonnet(baseDir string, env *v1alpha1.Config, opts jsonnet.Opts) (inter
 	}
 	opts.ExtCode.Set(spec.APIGroup+"/environment", string(jsonEnv))
 
-	// evaluate Jsonnet
-	mainFile := filepath.Join(baseDir, "main.jsonnet")
-	raw, err := jsonnet.EvaluateFile(mainFile, opts)
-	if err != nil {
-		return nil, err
+	var raw string
+	if opts.EvalPattern == "" {
```

Comment:
Nitpick: The logic seems inverted here. 

This read like "if no opts.EvalPattern, then run normally, but if not no opts.EvalPattern, then do EvalPattern".
I bit more clear would be "if opts.EvalPattern, then do EvalPattern, otherwise run normally:

```go
if opts.EvalPattern != "" {} else {}
```